doc/cephfs/administration.rst

```diff
@@ -237,6 +237,97 @@ For example, to only allow Nautilus clients, use:
 
 Clients running an older version will be automatically evicted.
 
+Enforcing minimum version of CephFS client is achieved by setting required
+client features. Commands to manipulate required client features of a file
+system:
+
+::
+
+    fs required_client_features <fs name> add luminous
+    fs required_client_features <fs name> rm luminous
```

Comment:
The goal of this PR is to move away from release names. It's fine that we have bits for older releases (with nebulous meaning) but we shouldn't encourage it further. Let's instead use one of these features below, like `deleg_ino`.